scripts/packaging/createupmpackages.ps1

```diff
@@ -105,49 +100,22 @@ if ($nodejsInstalled -eq $false)
 # Beginning of the upm packaging script main section
 # The overall structure of this script is:
 #
-# 1) Copy the appropriate publishing (.npmrc) file and set the npm command
-# 2) Replace the %version% token in the package.json file with the value of PackageVersion
-# 3) Overwrite the package.json file
-# 4) Create and publish the packages (local publishing copies the package to the OutputFolder)
-# 5) Cleanup files created and/or modified
-
-$npmrcFile = ".npmrc"
-$npmrcFileFullPath = "$scriptPath\$npmrcFile.$OutputTarget"
+# 1) Replace the %version% token in the package.json file with the value of PackageVersion
+# 2) Overwrite the package.json file
+# 3) Create and the packages and copy to the OutputFolder
+# 4) Cleanup files created and/or modified
+
 $cmdFullPath = "$env:systemroot\system32\cmd.exe"
-$npmCommand = "pack"
 $updateAuth = $true

```

Comment:
Do we need auth for packing?

Reply:
nope. good call. i will remove that part now